config/webpacker.yml

```diff
@@ -10,13 +10,17 @@ default: &default
 
   # Additional paths webpack should lookup modules
   # ['app/assets', 'engine/foo/app/assets']
-  additional_paths: []
+  additional_paths: [
+    'vendor',
+    'app/webpacker/css',
+    'engines/web/app/assets/stylesheets'
+  ]
```

Comment:
We haven't finished removing sprockets yet, and I think it would compile any assets in those default directories again (so we'll compile them twice) if we didn't move them.